packs/feats/absolve-sins.json

```diff
@@ -7,11 +7,11 @@
             "value": "action"
         },
         "actions": {
-            "value": 1
+            "value": 2
         },
         "category": "class",
         "description": {
-            "value": "<p>Your forgiveness is without limit, though the path to penitence can be painful, especially against those who have trespassed against your current mortal form. Spend a Mythic Point; each enemy within a @Template[type:emanation|distance:30] takes @Damage[14d6[mental]] damage (@Check[will|against:class-spell|basic] save against your class DC or spell DC, whichever is higher) as you force them to reflect upon their sins and misdeeds. Any creature who dealt damage to you or your hierophant since the end of your previous turn uses the outcome one degree of success worse than the result of their saving throw.</p>"
+            "value": "<p>Your forgiveness is without limit, though the path to penitence can be painful, especially against those who have trespassed against your current mortal form. Spend a Mythic Point; each enemy within a @Template[type:emanation|distance:30] takes @Damage[14d6[mental]|options:area-damage] damage (@Check[will|against:class-spell|basic|options:area-effect] save against your class DC or spell DC, whichever is higher) as you force them to reflect upon their sins and misdeeds.</p>\n<p>Any creature who dealt damage to you or your hierophant since the end of your previous turn uses the outcome one degree of success worse than the result of their saving throw.</p>"
```

Comment:
Did you intend to link to the Absolve Sins effect?

Reply:
I linked it and didn't push the change (: